src/main/resources/Journal.fxml

```diff
@@ -0,0 +1,85 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<?import java.lang.*?>
+<?import javafx.scene.control.*?>
+<?import javafx.scene.layout.*?>
+
+<Accordion maxHeight="-Infinity" maxWidth="-Infinity" minHeight="-Infinity" minWidth="-Infinity" prefHeight="1080.0" prefWidth="1920.0" xmlns="http://javafx.com/javafx/8" xmlns:fx="http://javafx.com/fxml/1">
+  <panes>
+    <TitledPane animated="false" prefHeight="389.0" prefWidth="600.0" text="Patient Profile">
+      <content>
+        <AnchorPane minHeight="0.0" minWidth="0.0" prefHeight="907.0" prefWidth="598.0">
+               <children>
+                  <TextField layoutX="16.0" layoutY="53.0" prefHeight="57.0" prefWidth="143.0" />
+                  <Label layoutX="14.0" layoutY="14.0" text="First Name:" />
+                  <Label layoutX="171.0" layoutY="14.0" text="Last Name:" />
+                  <TextField layoutX="176.0" layoutY="53.0" prefHeight="57.0" prefWidth="143.0" />
+                  <Label layoutX="331.0" layoutY="14.0" text="Age:" />
+                  <TextField layoutX="331.0" layoutY="53.0" prefHeight="57.0" prefWidth="83.0" />
+                  <Label layoutX="431.0" layoutY="14.0" text="Sex:" />
+                  <ChoiceBox layoutX="431.0" layoutY="54.0" prefWidth="150.0" />
+                  <Label layoutX="14.0" layoutY="114.0" text="Medical Insurance:" />
+                  <TextField layoutX="16.0" layoutY="153.0" prefHeight="57.0" prefWidth="304.0" />
+                  <TextField layoutX="331.0" layoutY="153.0" prefHeight="57.0" prefWidth="83.0" />
+                  <Label layoutX="331.0" layoutY="114.0" text="DOB:" />
+                  <Label layoutX="431.0" layoutY="114.0" text="Primary Doctor:" />
+                  <TextField layoutX="431.0" layoutY="153.0" prefHeight="57.0" prefWidth="172.0" />
+                  <Label layoutX="14.0" layoutY="234.0" text="Mailing Address:" />
+                  <Label layoutX="14.0" layoutY="274.0" text="Street:" />
+                  <TextField layoutX="16.0" layoutY="313.0" prefHeight="57.0" prefWidth="304.0" />
+                  <TextField layoutX="336.0" layoutY="313.0" prefHeight="57.0" prefWidth="304.0" />
+                  <Label layoutX="334.0" layoutY="274.0" text="Town:" />
+                  <ChoiceBox layoutX="651.0" layoutY="314.0" prefWidth="150.0" />
```

Comment:
I didn't use a choicebox. I used a menubutton, but I do think a choice box is a better option.